### PR TITLE
-Did some changes

### PR DESCRIPTION
**Pull request by AnnaT222**

### CODE REVIEW

File: src/MortgageApplicationQueueProcessor.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+class MortgageApplicationQueueProcessor {
+  customerRepository: CustomerRepository;
+
+  constructor(customerRepository: CustomerRepository) {
+    this.customerRepository = customerRepository;
+  }
+
+  static MESSAGE_INVALID_CUSTOMER: string = "Customer not found!";
```
Comment (Hades3110) [inline src/MortgageApplicationQueueProcessor.ts]: No need to explicitly assign the string type here, as TypeScript can infer it from the assigned value.

Comment (Hades3110) [reply src/MortgageApplicationQueueProcessor.ts]: I think this property may be private and read-only

File: src/MortgageApplicationQueueProcessor.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  processRequest(customerId: number, amountRequested: number): void {
+    this.updateBalance(customerId, amountRequested);
+  }
+  updateBalance(customerId: number, amountRequested: number): void {
+    const customer = this.getCustomer(customerId);
+    customer?.updateBalance(amountRequested);
+  }
+  getCustomer(customerId: number): NullableCustomer {
```
Comment (Hades3110) [inline src/MortgageApplicationQueueProcessor.ts]: Does this method return null ?

File: src/domain/Customer.ts [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+    updateBalance(amount:number):void {
        if(this.isEligibleForMortgage(amount)){
            this.balance += amount;
        }else{
            throw new NotEligibleForMortgageException();
        }
     }
 
-    isEligibleForMortgage(amountRequested) {
+    isEligibleForMortgage(amountRequested:number):boolean {
```
Comment (Hades3110) [inline src/domain/Customer.ts]: Make it private

**Review by Hades3110 — COMMENTED** [review]
make some internal methods private so that they cannot be called outside the class

File: src/MortgageApplicationQueueProcessor.ts (hunk 1 above)
Comment (AnnaT222) [reply src/MortgageApplicationQueueProcessor.ts]: Done!

File: src/MortgageApplicationQueueProcessor.ts (hunk 2 above)
Comment (AnnaT222) [reply src/MortgageApplicationQueueProcessor.ts]: No, you're right. Here should be the type of Customer cuz it's already checked with the method the line before. Corrected it.

File: src/domain/Customer.ts (hunk 3 above)
Comment (AnnaT222) [reply src/domain/Customer.ts]: Fixed!

**Pull request closed without merging**